### DISCUSSION TITLE
Writing to temp folder?

### DISCUSSION THREAD

**Discussion by appleb4** [AAndyProgram/SCrawler#193] [Q&A] [answered]
I noticed every time the downloader runs, a new folder is created in the windows appdata/local/temp folder. Usually something like "_MEI" followed by 6 numbers. Inside looks to be a bunch of .py files. I'm wondering, what part of Scrawler needs this, and is it possible to have this saved somewhere within the scrawler folder to re-use them?

**Comment by AAndyProgram** [comment, marked as answer]
These folders are created by gallery-dl ([which is what it is used for](https://github.com/AAndyProgram/SCrawler/wiki/Settings#gallery-dl)). You can simply delete them manually or using any automatic cleaning software (like CCleaner)

**Reply by appleb4** [reply]
> These folders are created by gallery-dl ([which is what it is used for](https://github.com/AAndyProgram/SCrawler/wiki/Settings#gallery-dl)). You can simply delete them manually or using any automatic cleaning software (like CCleaner)

Thank you, that's what I was thinking but wasn't sure.

**Reply by AAndyProgram** [reply]
**Join our Discord server**: https://discord.gg/uFNUXvFFmg
